README.md

```diff
@@ -783,73 +787,68 @@ Here are a few ways you might hear of or see in documents:
 ([usually](https://lsvp.wordpress.com/2008/09/15/what-entrepreneurs-need-to-know-about-founders-stock/))
 common stock allocated at a company’s formation, but otherwise doesn’t have any different
 rights from other common stock.
 
-Now some details on how this works:
+Some key insights on all this:
 
 - 🄳 Preferred stock usually has a
   [**liquidation preference (or preference)**](http://www.investopedia.com/terms/l/liquidation-preference.asp),
-  meaning the preferred stock owner will be paid before the common stock owners when a
-  [liquidity event](#sales-and-liquidity) happens, such as if the company is sold or IPOs.
+  meaning the preferred stock owners will be paid before the common stock owners when a
+  [liquidity event](#sales-and-liquidity) occurs, such as if the company is sold or goes public.
 - 🄳 A company is in
   [**liquidation overhang**](https://equityzen.com/blog/startup-valuations-and-liquidation-preference-overhang/)
-  when the value of a company doesn’t reach the number of dollars investors put into it.
+  when the value of the company doesn’t reach the dollar amount investors put into it.
   Because of liquidation preference, those holding preferred stock (investors) will have to
   be paid before those holding common stock (employees).
   If investors have put millions of dollars into your company and it’s sold, your equity as
   an employee
   [won’t be worth anything](https://avc.com/2010/10/employee-equity-the-liquidation-overhang/)
   if the company is in liquidation overhang and the sale doesn’t exceed that amount.
-- ☝️ Preferences are
-  [notoriously complex](https://venturebeat.com/2010/08/16/beware-the-trappings-of-liquidation-preference/).
-  We won’t cover all the details here, but generally, it’s worth understanding that
-  investors and entrepreneurs negotiate a lot of details, including:
-
+- ☝️ The complexities of the liquidation preference are
+  [infamous](https://venturebeat.com/2010/08/16/beware-the-trappings-of-liquidation-preference/).
+  It’s worth understanding that investors and entrepreneurs negotiate a lot of the details
+  around preferences, including:
   - The **multiple**, a number designating how many times the investor must be paid back before
-    common shareholders receive proceeds (often the multiple is 1X, but it can be 2X or
-    higher).
+    common shareholders receive proceeds.
+    (Often the multiple is 1X, but it can be 2X or higher.)
   - Whether preferred stock is
     [**participating**](https://en.wikipedia.org/wiki/Participating_preferred_stock), meaning
     investors get their money back and also participate in proceeds from common stock.
   - Whether there is a **cap**, which limits the payout if it is participating.
-
 - [This primer](https://medium.com/@CharlesYu/the-ultimate-guide-to-liquidation-preferences-478dda9f9332)
-  by Charles Yu gives a concise overview.
+  by Charles Yu gives a concise overview of preferred stock and the liquidation preference.
```

Comment:
the liquidation preference -> liquidation preferences

Reply:
Fixed. (Though the definite article refers to a concept in all its varied forms!)